### DISCUSSION TITLE
求助 下载的苹果IPA软件后缀变成zip格式！

### DISCUSSION THREAD

**Discussion by qijixiu** [AlistGo/alist#5508] [Q&A]
![QQ截图20231112164348](https://github.com/alist-org/alist/assets/84918228/857406a6-31fc-4082-89d3-ea400304ef63)

**Comment by kaixinxiaowangzi666** [comment]
改下后缀名就行了

**Comment by qijixiu** [comment]
苹果手机重命名的时候修改不了，谷歌浏览器跟电脑下载就没有问题，苹果自带浏览器就会，问了说下个版本会更新

发自我的iPhone

**Reply by kaixinxiaowangzi666** [reply]
那就等更新
